src/components/Calendar.tsx

```diff
@@ -14,35 +14,37 @@ const Calendar = ({ focusDate = new Date(Date.now()), events }: CalendarProps):
 
     useEffect(() => {
         const height = calendar.current?.clientHeight || 0;
-        const y = height / 24 * 8; // approx 8am
+        const y = height / 24 * 7; // approx 7am
         calendarWrapper.current?.scrollTo(0, y);
     }, []);
 
+    focusDate.setUTCHours(0,0,0,0);
```

Comment:
There's a problem with doing this, and I only thought of it as a temporary solution when we did it together earlier, sorry for not being clear about that.

The problem is that `setUTCHours()` mutates the date object, and the date object is not "owned" by this component but passed from outside. This means that we are actually modifying data outside of the component function by doing this, so the component/function is no longer [pure](https://en.wikipedia.org/wiki/Pure_function).

Imagine doing this:

```jsx
const rightNow = new Date();
console.log(rightNow); // Prints 2021-05-18 18:38 etc

<Calendar focusDate={ rightNow }/>

console.log(rightNow); // Prints 2021-05-18 00:00 etc
```

This is a very common source for bugs, which is why keeping functions pure is always preferable when possible.

A better way would have been to copy the object before mutating it, e.g.:

```jsx
const focusDateMidnight = new Date(focusDate);
focusDateMidnight.setUTCHours(0,0,0,0);
```

I'm making this comment because I think it's something to be vary of in general. But we don't need to do anything about it here, because as we discussed, we will soon be removing the `focusDate` property anyway and replacing it with something nicer once we've figured out more about the use case of the component on the calendar page.